### PR TITLE
Finalize extratorrent provider

### PR DESCRIPTION
**Pull request by miigotu**
@duramato

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

File: sickbeard/providers/extratorrent.py [hunk 1]
```diff
@@ -59,9 +58,6 @@ def __init__(self):
     def isEnabled(self):
         return self.enabled
 
-    def imageName(self):
-        return 'extratorrent.png'
```
Comment (duramato) [inline sickbeard/providers/extratorrent.py]: Why did you remove this?

Comment (miigotu) [inline sickbeard/providers/extratorrent.py]: @duramato it isnt needed as long as you name the file right.

That class is derived from `sickbeard.generic.TorrentProvider`, which is derived from `sickbeard.generic.GenericProvider`

If you look in `sickbeard.generic.GenericProvider` here:
https://github.com/SiCKRAGETV/SickRage/blob/master/sickbeard/providers/generic.py#L97

You see these three methods:
```python
   def getID(self):
        return GenericProvider.makeID(self.name)

    @staticmethod
    def makeID(name):
        return re.sub("[^\w\d_]", "_", name.strip().lower())

    def imageName(self):
        return self.getID() + '.png'
```

This takes the provider name, replaces anything that isn't a letter or number, strips whitespace from the ends, and lowercases it and then appends `.png`

Why override it if we dont need to?
